src/components/WebcamCapture/WebcamCapture.test.js

```diff
@@ -7,30 +7,24 @@ import Notification from './../Notification/Notification';
 
 configure({adapter: new Adapter()});
 
-const getProps = () => {
-	return {
-		onImgLoad: jest.fn(),
-		interval: 1
-	};
-};
+describe('<WebcamCapture />', () => {
 
-it("Tells the user that webcam isn't found if not found", () => {
-	const mockProps = getProps();
-	const wrapper = shallow(<WebcamCapture {...mockProps} />);
-	wrapper.setState({isDetecting: false});
-	expect(wrapper.find(Notification)).toHaveLength(1);
-});
+	it("Tells the user that webcam isn't found if not found", () => {
+		const wrapper = shallow(<WebcamCapture imgSize={300} />);
+		wrapper.setState({isDetecting: false});
+		expect(wrapper.find(Notification)).toHaveLength(1);
+	});
 
-it('Renders webcam with screenshot width of 300', () => {
-	const mockProps = getProps();
-	const wrapper = shallow(<WebcamCapture {...mockProps} />);
-	wrapper.setState({isDetecting: false, cameraConnected: true});
-	expect(wrapper.find(Webcam).props().screenshotWidth).toEqual(300);
-});
+	it('Renders webcam with screenshot width of 4/3 * imgSize', () => {
+		const wrapper = shallow(<WebcamCapture imgSize={300} />);
+		wrapper.setState({isDetecting: false, cameraConnected: true});
+		expect(wrapper.find(Webcam).props().screenshotWidth).toEqual(300 * 4/3);
```

Comment:
probably put 400 instead of 300 * 4/3
just so you are testing this maths bit too